kong/plugins/standard-webhooks/handler.lua

```diff
@@ -0,0 +1,12 @@
+local plugin = require "kong.plugins.standard-webhooks.internal"
+
+local StandardWebhooks = {
+    VERSION = require("kong.meta").version,
```

Comment:
Seems we should use "core_version" like this:

`KeyAuthHandler.VERSION = meta.core_version`

Reply:
https://github.com/Kong/kong/blob/master/kong/meta.lua there's no attribute `core_version` in meta.lua

Reply:
@zekth ha, you are right. On CE, we jus use `.version`. When cherry-picking to EE, we should use `.core_version`. See https://github.com/Kong/kong-ee/blob/1a13abcc3588077842d8be0d539dfa6ae14931a8/kong/meta.lua#L19